.github/workflows/go-test.yml

```diff
@@ -19,7 +19,6 @@ jobs:
           - ubuntu-latest
           - macos-latest
         go-version:
-          - 1.22.x
           - 1.23.x
```

Comment:
We can add 1.24 here, released today :slightly_smiling_face: This is probably why these tools have dropped support for 1.22.
```suggestion
          - 1.23.x
          - 1.24.x
```

Reply:
ooh sure!